### PR TITLE
Jenkins Integration with Kabanero reference doc on kabanero.io

### PR DESCRIPTION
**Pull request by dewan-ahmed**
Please fix the reference to sample_jenkinsfile under "Reference" section. the sample_jenkinsfile is attached in the same directory.

### CODE REVIEW

**Comment by alohr51** [comment]
hi @dewan-ahmed thanks for the contribution! For the images take a look at https://github.com/kabanero-io/docs#add-images and update your images in your PR -  let me know if you have any questions.

File: ref/general/sample_jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        stage('Deploy') {
+            container('kubectl') {
+                checkout scm
+                sh 'sed -i -e \'s#applicationImage: .*$#applicationImage: docker-registry.default.svc:5000/Kabanero/project1#g\' app-deploy.yaml'
+                sh 'cat app-deploy.yaml'
+                sh 'find . -name app-deploy.yaml -type f|xargs kubectl apply -f'
+            }
+        }
+    }
+}
```
Comment (alohr51) [inline ref/general/sample_jenkinsfile]: each file in ref/general needs to be a full and complete doc. If you want it included in `/jenkins-integration-with-kabanero.adoc `you'll need to place it in that doc directly

File: ref/general/jenkins-integration-with-kabanero.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
+:page-layout: doc
+:page-doc-category: Reference
+:page-title: Jenkins Integration with Kabanero
+:sectanchors:
+= Jenkins Integration with Kabanero
+
+The goal of this guide is to show existing Jenkins users how to use Tekton to build images from their code repository and have Jenkins deploy using the image. The flow looks something like this:
+
+image::jenkins-tekton.png[]
```
Comment (alohr51) [inline ref/general/jenkins-integration-with-kabanero.adoc]: need to update image placement and link, please see https://github.com/kabanero-io/docs#add-images

**Comment by alohr51** [comment]
@dewan-ahmed is this supposed to be a guide? If so this is not the repo to put it in. This is for documentation (not guides)

For info on contributing guides take a look at https://github.com/kabanero-io/draft-guides-template

**Comment by dewan-ahmed** [comment]
@alohr51 per Akihiko, this is supposed to go under Reference section of Docs. Also added images according to the instruction. Please review

**Comment by dewan-ahmed** [comment]
This was pushed under Docs based on the initial content-request. Now working on the changes to put it under Guide section.

**Pull request closed without merging**